build-and-test.ps1

```diff
@@ -0,0 +1,24 @@
+[cmdletbinding()]
+param(
+   [string]$OS="windowsservercore"
+)
+
+Set-StrictMode -Version Latest
+$ErrorActionPreference="Stop"
+
+$dockerRepo="microsoft/dotnet-nightly"
+
+pushd $PSScriptRoot
+
+Get-ChildItem -Recurse -Filter Dockerfile | where DirectoryName -Ilike "*\$OS*" | sort DirectoryName | ForEach {
+    $tag = "$($dockerRepo):" + $_.DirectoryName.Replace($PSScriptRoot, '').TrimStart('\').Replace('\', '-') -replace "windowsservercore$", "windowsservercore-sdk"
```

Comment:
I'm wondering if it would be better to use the $OS variable rather than hard-coding windowsservercore, because at this point, the "-sdk" tagging convention is the same for all OSes.

`-replace "${OS}$", "${OS}-sdk"`